src/error.rs

```diff
@@ -72,7 +72,7 @@ pub enum Error {
     /// Bee common logger error
     #[error("{0}")]
     #[serde(serialize_with = "display_string")]
-    CommonError(#[from] bee_common::logger::Error),
+    CommonError(#[from] fern_logger::Error),
```

Comment:
Should we call it `LoggerError` or `FernError` then?

Reply:
Whoops, missed this, will do